### PR TITLE
add workflow status rest  endpoint

### PR DESCRIPTION
**Pull request by anludke**

### CODE REVIEW

File: workflow-service/src/main/java/com/redhat/parodos/workflow/execution/service/WorkFlowServiceImpl.java [hunk 1]
```diff
@@ -130,6 +139,140 @@ public synchronized WorkFlowExecution updateWorkFlow(WorkFlowExecution workFlowE
 		return workFlowRepository.save(workFlowExecution);
 	}
 
+	@Override
+	public WorkFlowStatusResponseDTO getWorkFlowStatus(UUID workFlowExecutionId) {
```
Comment (pkliczewski) [inline workflow-service/src/main/java/com/redhat/parodos/workflow/execution/service/WorkFlowServiceImpl.java:143]: This method seems to be long. Do you think it makes sense to split it?

**Review by pkliczewski — COMMENTED** [review]
overall looks good, 2 small comments

File: workflow-service/src/main/java/com/redhat/parodos/workflow/execution/service/WorkFlowServiceImpl.java [hunk 2]
```diff
@@ -130,6 +139,140 @@ public synchronized WorkFlowExecution updateWorkFlow(WorkFlowExecution workFlowE
...
+								? com.redhat.parodos.workflow.enums.WorkStatus.PENDING
+								: com.redhat.parodos.workflow.enums.WorkStatus
+										.valueOf(masterWorkFlowExecution.getStatus().name()))
+						.numberOfWorks(masterWorkFlowDefinition.getNumberOfWorks()).works(new ArrayList<>()).build());
+		workFlowWorksStartIndex.put(masterWorkFlowDefinition.getName(), 1);
+
+		List<WorkFlowWorkDefinition> masterWorkFlowWorkDefinitions = workFlowWorkRepository
+				.findByWorkFlowDefinitionIdOrderByCreateDateAsc(masterWorkFlowDefinition.getId());
+		masterWorkFlowWorkDefinitions.forEach(workFlowWorkDefinition -> {
+			if (workFlowWorkDefinition.getWorkDefinitionType().equals(WorkType.WORKFLOW)) {
```
[the remark this answers is not in the document]
Comment (anludke) [reply workflow-service/src/main/java/com/redhat/parodos/workflow/execution/service/WorkFlowServiceImpl.java:171]: @eloycoto I believe it's the same comment as @pkliczewski.

**Comment by pkliczewski** [comment]
/lgtm
We agree with @anludke to improve the method in the next PR

**Pull request merged**